src/data-import/markdown-file.ts

```diff
@@ -304,6 +307,42 @@ export function parseLists(
     return [Object.values(cache), literals];
 }
 
+/**
+ * Parse table from content and meta data. The implementation will not include column spans. It only work for
+ * one column/one cell value.
+ */
+export function parseTables(
+    content: string[],
+    metadata: CachedMetadata,
+): Map<string, any>[] {
+    const result: Map<string, any>[] = [];
+    metadata.sections?.forEach(section => {
+        if (section.type === 'table') {
+            const headers = content[section.position.start.line]
+                .split('|')
+                .map(header => header.trim())
+                .filter(header => header);
+            let rows = content
+                .slice(section.position.start.line + 2, section.position.end.line + 1)
+                .map(row => row
+                    .split('|')
+                    .map(val => val.trim())
+                    .filter(row => row));
+            // don't support expand column at the moment.
+            if (rows.length === headers.length) {
```

Comment:
Should probably have to be `rows[0].length`; it currently checks whether the table has the same number of rows as columns. To test this bug, consider adding another row to `test-vault/tables/git commands.md`

```
| `git log` | Show commit logs |
```

Reply:
Oh, nevermind! I was looking at an earlier commit. :-)